tests/acceptance/features/bootstrap/WebUIUsersContext.php

```diff
@@ -50,6 +50,12 @@ class WebUIUsersContext extends RawMinkContext implements Context {
 	 */
 	private $webUIGeneralContext;
 
+	/**
+	 *
+	 * @var GroupsCreated
+	 */
+	private $groupsCreated = [];
```

Comment:
use the created groups from featureContext see https://github.com/owncloud/core/blob/5fff7599545af9e365eda39af3c52798f0517208/tests/acceptance/features/bootstrap/Provisioning.php#L42
there is no `add` function so you will have to create one there